src/courses/mappers/18.md

```diff
@@ -0,0 +1,271 @@
+---
+order: 18
+title: Appendix - SAF CLI Integration
+author: Charles Hu
+---
+
+## SAF CLI Integration
+
+The following is a supplemental lesson on how to integrate your developed mapper with the SAF CLI. Integration with the SAF CLI will allow your mapper to be used in a command line environment, independent of Heimdall2.
+
+If you have not yet created a mapper, please follow the primary course and do so. Once completed, you will be able to continue with this lesson.
+
+### Set Up
+
+First, we need set up the necessary files to begin integrating your mapper with the SAF CLI.
+
+1. Create a development branch against the [SAF CLI repository](https://github.com/mitre/saf) and create a draft pull request for your new branch.
+
+2. In the `package.json` file, update the versions of `@mitre/hdf-converters` and `@mitre/heimdall-lite` to the [latest release of Heimdall2](https://github.com/mitre/heimdall2/releases).
```

Comment:
have a little drop down section here to explain how to use the commands to use an in-development mapper so you don't need to wait for the mapper to be merged in and released before making a saf cli integration

Reply:
I'm blanking on the exact command, could you remind what you're talking about?